Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build install test integration-test clean dev-setup watch release-build release-clean lint lint-install lint-fix 
+.PHONY: build install test integration-test clean dev-setup watch release-build release-clean lint lint-install lint-fix setup-worktree
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)